cv_bridge/CMakeLists.txt

```diff
@@ -3,16 +3,18 @@ project(cv_bridge)
 
 find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
+find_package(Boost QUIET)
 if(NOT ANDROID)
-  find_package(PythonLibs)
-  if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
-    find_package(Boost REQUIRED python)
+  if(Boost_VERSION LESS 106700)
+    set(BOOST_PYTHON_COMPONENT_SUFFIX "-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
```

Comment:
The `Boost_VERSION LESS 106700` case looks different from the proposal in #239. I was wondering if that works also on macos with brewed python. But I guess that doesn't matter, since brewed boost is always > 106700 nowadays.

Reply:
Yeah, I think the main thing for old-Boost is just working with Ubuntu happens to do, which is this.